redfish-core/lib/fabric_adapters.hpp

```diff
@@ -196,6 +224,15 @@ inline void doAdapterGet(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
     getFabricAdapterAsset(asyncResp, serviceName, fabricAdapterPath);
     getFabricAdapterState(asyncResp, serviceName, fabricAdapterPath);
     getFabricAdapterHealth(asyncResp, serviceName, fabricAdapterPath);
+
+    // if the adapter also implements this interface, link the adapter schema to
+    // PCIeDevice schema for this adapter.
+    if (std::find(interfaces.begin(), interfaces.end(),
```

Comment:
Similar to `std::sort()` to `std::ranges::sort()`, this can be written using `std::ranges::find()`.
```
    if (std::ranges::find(interfaces, "xyz.openbmc_project.Inventory.Item.PCIeDevice") !=
```